aws/resource_aws_dlm_lifecycle_policy.go

```diff
@@ -65,7 +65,7 @@ func resourceAwsDlmLifecyclePolicy() *schema.Resource {
 												"interval": {
 													Type:     schema.TypeInt,
 													Required: true,
-													ValidateFunc: validateIntegerInSlice([]int{
+													ValidateFunc: validation.IntInSlice([]int{
```

Comment:
Minor nit the total amount of Int values for this slice is small enough to have on a single line. Let’s inline the values so that it’s easier to grok.

Reply:
@nywilken change added